docs/components/Typography/Typography.stories.mdx

```diff
@@ -0,0 +1,76 @@
+import { Canvas, Meta } from "@storybook/addon-docs";
+import { Typography } from "@jobber/components/Typography";
+import { Banner } from "@jobber/components/Banner";
+
+# Typography
+
+<Meta title="Components/Text and Typography/Typography/Docs" />
+
+Base component that defines typographic standards across Jobber.
+
+<Banner type="notice" dismissible={false}>
+
+Typography is the component that powers our Heading and Text components “under
+the hood”. You should first look to use
+[Heading](../?path=/docs/components-text-and-typography-heading--heading) and
+[Text](../?path=/docs/components-text-and-typography-text-docs--page) before you
+reach for Typography.
+
+</Banner>
+
+## Font size
```

Comment:
Maybe can we have a sentence here linking to the Design Typography section where we have the web and mobile size tokens?

Reply:
should be (../?path=/docs/design-typography--page#font-size)

Reply:
Added it! good idea